docs/architecture/distributed-cloud-native-apps-containers/introduction-to-cloud-native-development/what-is-cloud-native.md

```diff
@@ -0,0 +1,344 @@
+| [Security](https://learn.microsoft.com/azure/well-architected/reliability/) | Implement security across the entire lifecycle of an application, from design and implementation to deployment and operations. Pay close attention to identity management, infrastructure access, application security, and data sovereignty and encryption. |
+
+To get started, Microsoft provides a set of [online assessments](https://learn.microsoft.com/assessments/azure-architecture-review/) to help you assess your current cloud workloads against the five well-architected pillars.
+
+## Microservices
+
+Cloud-native systems embrace microservices, a popular architectural style for constructing modern applications.
+
+Built as a distributed set of small, independent services that interact through a shared fabric, microservices share the following characteristics:
+
+- Each implements a specific business capability within a larger domain context.
+
+- Each is developed autonomously and can be deployed independently.
+
+- Each is self-contained encapsulating its own data storage technology, dependencies, and programming platform.
+
+- Each runs in its own process and communicates with others using standard communication protocols such as HTTP/HTTPS, gRPC, WebSockets, or [AMQP](https://en.wikipedia.org/wiki/Advanced_Message_Queuing_Protocol).
+
+- They compose together to form an application.
+
+Figure 1-4 contrasts a monolithic application approach with a microservices approach. Note how the monolith is composed of a layered architecture, which executes in a single process. It typically consumes a relational database. The microservice approach, however, segregates functionality into independent services, each with its own logic, state, and data. Each microservice hosts its own datastore.
+
+![Monolithic deployment versus microservices](./media/monolithic-vs-microservices.png)
+
+**Figure 1-4.** Monolithic versus microservices architecture
+
+Note how microservices promote the **Processes** principle from the [Twelve-Factor Application](https://12factor.net/), discussed earlier in the chapter.
+
+> *Factor \#6  specifies "Each microservice should execute in its own process, isolated from other running services."*
+
+### Why microservices?
+
+Microservices provide agility.
+
+Earlier in the chapter, we compared an eCommerce application built as a monolith to that with microservices. In the example, we saw some clear benefits:
+
+- Each microservice has an autonomous lifecycle and can evolve independently and deploy frequently. You don't have to wait for a quarterly release to deploy a new feature or update. You can update a small area of a live application with less risk of disrupting the entire system. The update can be made without a full redeployment of the application.
+
```

Comment:
You might want to mention that each microservice is often written by a dediicated and independent team.